### ISSUE TITLE
when will the Mask-Rcnn be implemented

### ISSUE THREAD

**Issue by puppy2000** [MIC-DKFZ/nnDetection#133]
### :question: Question

hello,because medicaldetectiontoolkit has been deprecated,I want to know when will the Mask-Rcnn 3d version be integrated into this framework,Looking forward to it.Thank you very much!

**Comment by Da1daidaidai** [comment]
same question,  looking forward to this update:)

**Comment by mibaumgartner** [comment]
Sorry for the delay, I was on vacation.
We don't have a specific ETA yet but we are currently finalising our results and hope to write everything up soonish (2-3 months).

**Comment by dalbis** [comment]
Hi! today I was going to start writing some code to use Mask-RCNN in nndetection, then I found this thread. Now I am not sure I should make the effort or perhaps just wait. Is this still planned to be released soonish? Thank you!

**Comment by mibaumgartner** [comment]
Yes, still planning to add R-CNN models, it will take a bit more time though since another project took more time than expected (still stuck at ~2 month)

**Comment by vincentwan0930** [comment]
same question, looking forward to this update:). 
If i want to modify the modules of retina UNet or replace it by another network, I want to know which file and which part should i modify?

**Comment by khaghanijavad** [comment]
Hi, I have the same question. Are you planning to integrate Mask R-CNN into nnDetection? Thank you.
